azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+trigger:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - azure-pipelines.yml
+    - 'Fakebook.Profile'
+
+pr:
+- 'main'
```

Comment:
we want PRs to the user story branches to be built also, if not all PRs period.
and there's no reason for the path trigger to differ between PR and non-PR builds